lab-cameron/test/sll-test.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const Node = require('../model/list-node.js');
+const SLL = require('../lib/sll.js');
+const mocha = require('mocha');
```

Comment:
**TODO FOR LOST POINTS**
No need to require mocha in like this, it's something that gets grabbed in as a dependency via your package.json. (points off because it doesn't pass Linter because it's redlined for being an unused variable).